dev_scripts/upload.sh

```diff
@@ -7,8 +7,14 @@
 
 BUGSNAG_URL=https://upload.bugsnag.com/react-native-source-map
 BUGSNAG_KEY=16a680e189b1e5f03f28665870f1401f
-APP_VERSION=3.2.4
-
+APP_VERSION=$(node -e "console.log(require('./package.json').version)")
+if [ -z "APP_VERSION" ] # if unable to obtain version
```

Comment:
remove quotes please

Reply:
@andreievg 
It doesn't work if quotes been removed!
Extra code has been removed, see `10a1f6a22`

Reply:
Ok, the PR looks good now, but I was wondering, since you've written the if statement code, what do you think was wrong with the if statement ?